src/extensions/ReinforcementLearningBase.jl

```diff
@@ -15,13 +13,12 @@ Random.rand(s::MultiContinuousSpace{<:CuArray}) = rand(CuArrays.CURAND.generator
 Flux.testmode!(p::AbstractPolicy, mode = true) =
     @error "someone forgets to implement this method!!!"
 
-macro policy_str(path)
-    load_policy(path)
+function save(f::String, p::AbstractPolicy)
+    policy = cpu(p)
+    BSON.@save f policy
 end
 
-function load_policy(path)
-    if isdir(path)
-        path = joinpath(path, "policy.bson")
-    end
-    BSON.load(path)[:policy]
-end
+function load(f::String, ::Type{<:AbstractPolicy})
+    BSON.@load f policy
+    gpu(policy)
```

Comment:
In most cases, calling `gpu` is useful. But not sure if it should be done implicitly.